### PR TITLE
update readme version

### PR DESCRIPTION
**Pull request by sriv**

### CODE REVIEW

**Comment by zabil** [comment]
I did a bit of debugging to isolate the issue. I was able to setup up an Ubuntu instance on my Mac using [Multipass](https://multipass.run) as the test were failing on Ubuntu.

The issue is with shutting down the server at
https://github.com/getgauge/gauge-lsp-tests/blob/82d0168207cea88d52f4daf88ec64d44b1a21a55/tests/project.js#L30-L32

This fails every time.

The lsp logs for a normal exit (on Windows and Mac) is

```
21-04-2022 22:46:24.414 [Gauge] [DEBUG] jsonrpc2: --> request #3: shutdown: null
21-04-2022 22:46:24.416 [dotnet] [DEBUG] KillProcessrequest received
21-04-2022 22:46:24.518 [Gauge] [DEBUG] Runner with PID:15351 has exited
21-04-2022 22:46:24.518 [Gauge] [DEBUG] jsonrpc2: <-- result #3: shutdown: null
21-04-2022 22:46:24.519 [Gauge] [DEBUG] jsonrpc2: --> notif: exit: null
```

While on Ubuntu is

```
22-04-2022 14:09:02.855 [Gauge] [DEBUG] jsonrpc2: --> request #3: shutdown: [null]
22-04-2022 14:09:02.856 [dotnet] [DEBUG] KillProcessrequest received
22-04-2022 14:09:02.870 [Gauge] [DEBUG] jsonrpc2: <-- error #3: shutdown: {"code":0,"message":"rpc error: code = Unavailable desc = error reading from server: EOF","data":null}
```

For some reason a proper shutdown does not happens on Dotnet at Ubuntu

https://github.com/getgauge/gauge-dotnet/blob/7ee8f7e8bfdb09c380c3d37ff2e797e8a6adfb3e/src/AuthoringRunnerServiceHandler.cs#L111-L117

**Comment by sriv** [comment]
This could be related - https://github.com/grpc/grpc-dotnet/issues/1649#issuecomment-1081854085

File: src/AuthoringRunnerServiceHandler.cs [hunk 1]
```diff
@@ -111,12 +111,12 @@ public override Task<Empty> Kill(KillProcessRequest request, ServerCallContext c
             try
             {
                 Logger.Debug("KillProcessrequest received");
-                lifetime.StopApplication();
                 return Task.FromResult(new Empty());
             }
             finally
             {
                 _pool.Dispose();
+                Task.Delay(500).ContinueWith((_) => lifetime.StopApplication());
```
Comment (zabil) [inline src/AuthoringRunnerServiceHandler.cs:119]: From https://github.com/grpc/grpc-dotnet/issues/1649#issuecomment-1081854085

> This is a pretty low-level change in HTTP/2. Given the risk, for now, the fix will be in .NET 7 and later. We'll reevaluate backporting if more people are impacted by this.

Maybe it's worth waiting for the fix?

**Comment by sriv** [comment]
So I added a condition to ignore the error for Linux+dotnet in the LSP tests, but it turns out that this is an issue in Windows as well. I think this needs more investigation. Will look into this

**Pull request merged**